taxonomy/management/commands/refresh_course_skills.py

```diff
@@ -18,10 +18,12 @@
 
 class Command(BaseCommand):
     """
-        Example usage:
-            $ ./manage.py refresh_course_skills --course 'Course1_uuid' --course 'Course2_uuid' --commit
-            $ ./manage.py refresh_course_skills --args-from-database
-        """
+    Command to referesh skills associated with the courses.
+
+    Example usage:
+        $ ./manage.py refresh_course_skills --course 'Course1_uuid' --course 'Course2_uuid' --commit
```

Comment:
Shouldn't these be course ids instead of uuid?

Reply:
No, the command expected course UUIDs as arguments